data/USPTO_500k/meta.yaml

```diff
@@ -0,0 +1,40 @@
+name: USPTO_500k
+description: United States Patent and Trademark Office reaction dataset with yields.
+targets:
+- id: yield
+  description: Reaction yields analyzed by UPLC
+  units: '%'
+  type: continuous
+  names:
+  - Reaction yield
+  - yield
+  uris:
+  - https://bioportal.bioontology.org/ontologies/AFO?p=classes&conceptid=http%3A%2F%2Fpurl.allotrope.org%2Fontologies%2Fquality%23AFQ_0000227
+  - https://en.wikipedia.org/wiki/Yield_(chemistry)
```

Comment:
I would use the "id" in the ontology table, but I can show you at an example when we discuss

Reply:
@kjappelbaum  I'm not sure what you mean here?